SS14.Launcher/ViewModels/MainWindowTabs/ServerEntryViewModel.cs

```diff
@@ -227,7 +247,14 @@ private void OnCacheDataOnPropertyChanged(object? _, PropertyChangedEventArgs ar
             case nameof(IServerStatusData.Description):
             case nameof(IServerStatusData.StatusInfo):
                 OnPropertyChanged(nameof(Description));
+                OnPropertyChanged(nameof(HaveData));
                 break;
         }
     }
+
+    private static string GetTimeStringSince(DateTime dateTime)
+    {
+        var ts = DateTime.Now.ToUniversalTime().Subtract(dateTime);
+        return ts.Hours > 0 ? $"{ts.Hours}H {ts.Minutes}M" : $"{ts.Minutes}M";
```

Comment:
This isn't localizable.

Reply:
I did confirm that this same format isn’t being translated in the game (though there is a translation key), but I can add this as a key for future-proofing.

Reply:
Yeah I wouldn't have passed that through review in the main game either.